src/rosdep2/main.py

```diff
@@ -153,7 +153,10 @@ def rosdep_main(args=None):
     except UsageError as e:
         print(_usage, file=sys.stderr)
         print('ERROR: %s' % (str(e)), file=sys.stderr)
-        sys.exit(os.EX_USAGE)
+        if hasattr(os, 'EX_USAGE'):
+            sys.exit(os.EX_USAGE)
+        else:
+            sys.exit()
```

Comment:
Should this be a non-zero exit code to indicate an error even on platforms where EX_USAGE is not defined?

Reply:
Yes, I believe so. @traversaro - you are the Windows guru, is there a standard way of doing so or is simply "1" sufficient?

Reply:
`os.EX_USAGE` returns 64 on Unix, so I decided to return the same non-zero value on Windows.